client/context/EditorContext.tsx

```diff
@@ -9,6 +9,7 @@ export interface ObjectDetails {
     models: Array<string>;
```

Comment:
This file is getting large. Maybe we could create and import modules that concern children states, actions and datastream states, actions into this file. It's something to think about in the future and doesn't have to be necessarily addressed in this pr. I think I also made a mistake of inserting the snackbar actions into the file when that behavior could be useful for all pages of the application.

Reply:
Yes, agreed. I'll capture a task to refactor this context as a future task.